groups/sig-docs/groups.yaml

```diff
@@ -17,14 +17,13 @@ groups:
       WhoCanPostMessage: "ANYONE_CAN_POST"
       ReconcileMembers: "true"
     owners:
-      - jorgec@vmware.com
-      - kaitlynbarnard10@gmail.com
-      - paris.pittman@gmail.com
       - killen.bob@gmail.com
-      - zcorleissen@linuxfoundation.org
-    members:
-      - codyclark@google.com
-      - vonguard@gmail.com
+      - onlydole@gmail.com
+      - tim+cncf@scalefactory.com
+    # members:
+    #   -
+    #   -
+
```

Comment:
@nate-double-u you are an editor - now - so please do feel welcome to add yourself.